response_operations_ui/exceptions/error_codes.py

```diff
@@ -0,0 +1,26 @@
+from enum import Enum
+
+
+class ErrorCode(Enum):
```

Comment:
Introduced this so we don't have a proliferation of string literals through the new cir client. We should do more of this for consistency throughout the application.